app/middleware/renderer.js

```diff
@@ -1,7 +1,51 @@
+const log = require('../lib/logger');
+const messages = require('../lib/messages');
+
 function findHelp(req, res) {
   res.render('find-help');
 }
 
+function location(req, res) {
+  if (res.locals.locationHeading === null) {
+    return res.render('error');
+  }
+  return res.render('location');
+}
+
+function results(req, res) {
+  res.render('results');
+}
+
+function emptyPostcodePage(req, res) {
+  log.debug('Empty Search');
```

Comment:
Logs are best used when there is additional information to capture. It would be better to do something like increment a Prometheus counter.

Reply:
Will keep that in mind when we add metrics to the app.